lib/src/video_player/method_channel_video_player.dart

```diff
@@ -87,6 +87,7 @@ class MethodChannelVideoPlayer extends VideoPlayerPlatform {
           'maxCacheFileSize': dataSource.maxCacheFileSize,
           'cacheKey': dataSource.cacheKey,
           'showNotification': dataSource.showNotification,
+          'isLiveStream': dataSource.isLiveStream,
```

Comment:
memo:
The flag `isLiveStream ` is not passed to the native side at the current library, so need to implement it in this PR.